### PR TITLE
Migrate to yaml manifest

### PR DESCRIPTION
**Pull request by AsavarTzeth**
Flatpak 0.10.10 introduces support for manifests written in YAML, as an
alternative to JSON.

Using YAML makes the manifest a lot easier to read, many lines shorter
and decreases the risk of typos.

### CODE REVIEW

**Comment by AsavarTzeth** [comment]
bot, build

**Comment by AsavarTzeth** [comment]
bot, build

**Comment by AsavarTzeth** [comment]
I guess I tried a bit too early lol :)

https://flathub.org/builds/#/builders/2/builds/1060

```
flatpak-builder -v --force-clean --sandbox --delete-build-dirs --user --install-deps-from=flathub --arch x86_64 --bundle-sources --mirror-screenshots-url=https://flathub.org/repo/screenshots --repo repo --extra-sources=/mnt/builder/worker/build-x86_64-4/../downloads --default-branch stable --subject 'Migrate to yaml manifest (ac5d4d7c)' builddir com.github.wwmm.pulseeffects.json
 in dir /mnt/builder/worker/build-x86_64-4/com.github.wwmm.pulseeffects (timeout 3600 secs)
 watching logfiles {}
 argv: ['flatpak-builder', '-v', '--force-clean', '--sandbox', '--delete-build-dirs', '--user', '--install-deps-from=flathub', '--arch', 'x86_64', '--bundle-sources', '--mirror-screenshots-url=https://flathub.org/repo/screenshots', '--repo', 'repo', '--extra-sources=/mnt/builder/worker/build-x86_64-4/../downloads', '--default-branch', 'stable', '--subject', 'Migrate to yaml manifest (ac5d4d7c)', 'builddir', 'com.github.wwmm.pulseeffects.json']
 using PTY: True
Can't load 'com.github.wwmm.pulseeffects.json': Failed to open file '/mnt/builder/worker/build-x86_64-4/com.github.wwmm.pulseeffects/com.github.wwmm.pulseeffects.json': No such file or directory
program finished with exit code 1
elapsedTime=0.015904
```

**Comment by TingPing** [comment]
I imagine builders will be updated next week.

File: com.github.wwmm.pulseeffects.yml [hunk 1]
```diff
@@ -168,7 +161,13 @@ modules:
           - type: patch
             path: patch/gst-plugins-bad/0003-meson-enable-bs2b-compilation.patch
         post-install:
-          - install -Dm644 -t /app/share/licenses/gst-plugins-bad COPYING COPYING.LIB
+          - >
```
[the remark this answers is not in the document]
Comment (TingPing) [reply com.github.wwmm.pulseeffects.yml]: http://yaml-multiline.info/

[the remark this answers is not in the document]
Comment (AsavarTzeth) [reply com.github.wwmm.pulseeffects.yml]: Thank you, but I had already used that site to refresh my memory. It doesn't really change anything. I think you're confused by the fact that I moved line 171 to the bottom or something.

To be clear, changing to `|` for line 164 results in this:

```
install: missing file operand
Try 'install --help' for more information.
/bin/sh: line 1:     5 Segmentation fault      (core dumped) ext/bs2b/libgstbs2b.so
/bin/sh: line 2:     6 Segmentation fault      (core dumped) ext/ladspa/libgstladspa.so
/bin/sh: line 3:     7 Segmentation fault      (core dumped) ext/lv2/libgstlv2.so
```

[the remark this answers is not in the document]
Comment (AsavarTzeth) [reply com.github.wwmm.pulseeffects.yml]: Oh wait I see, I don't need `>` at all. Is that maybe it?

**Comment by AsavarTzeth** [comment]
Better now? In retrospect, I can see how your proposed style guide would be useful here.

File: com.github.wwmm.pulseeffects.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (TingPing) [reply com.github.wwmm.pulseeffects.yml]: Oh, yea you are totally right. I thought they were separate commands for some reason.

**Comment by TingPing** [comment]
Yea I do find this style more readable.

**Comment by AsavarTzeth** [comment]
bot, build

**Comment by AsavarTzeth** [comment]
bot, build

**Pull request merged**